### DISCUSSION TITLE
How in PIMCORE to synchronize the filling of the values ​​of product characteristics when maintaining a multilingual catalog?

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#8292] [Q&A / Support]
PimCore has created a skeleton for a product card.

For the card, two objectbrics for maintaining characteristics and values ​​have been created. In Russian and English.
Both objectBrics are identical in terms of characteristics and meanings (the only difference is in spelling, in Russian and English, respectively)

Everything works, and everything can be filled in, but there is a probable problem - the values ​​of the characteristics when manually filled in in different languages ​​can be different (human factor).

Is it possible, and how, to make sure that when choosing a value in the Russian version, for example, color ("чёрный"), the value "black" is automatically substituted in the English filling?

**Comment by passioneight** [comment]
That really depends on your application.

For example, I'd rather use localized fields instead of object bricks, unless the bricks' fields vary - which they do not, as per your description.

However, `color` is a field that sounds like a `select`, rather than a `text` field. So, I'd probably use a `select` with an `OptionsProvider`, using translatable options. [Here's our bundle](https://github.com/passioneight/pimcore-options-providers) to help you get started.

With this approach neither bricks nor localized fields are necessary - at least not for the field `color`. Additionally, you only have to set the correct value once.
